extensions/data-transfer/portability-data-transfer-google/src/main/java/org/dataportabilityproject/datatransfer/google/tasks/GoogleTasksExporter.java

```diff
@@ -106,7 +107,9 @@ public GoogleTasksExporter(GoogleCredentialFactory credentialFactory) {
         result
             .getItems()
             .stream()
-            .map(t -> new TaskModel(resource.getId(), t.getTitle(), t.getNotes()))
+            .map(t -> new TaskModel(resource.getId(), t.getTitle(), t.getNotes(),
+                Instant.ofEpochMilli(t.getCompleted().getValue()),
```

Comment:
What if t.getCompleted() or getDue() are null? Should we write some defensive code when fetching values?

Reply:
Added ternary operators.